extraction/extractor.py

```diff
@@ -84,8 +85,10 @@ def __extract_data_from_html_body(self, node):
     def __is_content(self, node):
         """This function will extract valid content and return it"""
 
-        # This function will be updated in next iteration
-        _ = node
         _ = self.soup
+        # checks if content is video
+        is_content, video_content = video_extractor.is_video(node)
+        if is_content:
+            return True, video_content
```

Comment:
Move video_Extractor to list. And loop through the lists.

Reply:
Changed is_video to validate_and_extract method of class VideoExtractor in video_extractor.

Reply:
Added tuple for IContentExtractor and updated accordingly.